src/phoenix/server/api/subscriptions.py

```diff
@@ -166,8 +168,51 @@ async def chat_completion_create(
     @abstractmethod
     def attributes(self) -> Dict[str, Any]: ...
 
-
-@register_llm_client(GenerativeProviderKey.OPENAI)
+    @classmethod
+    def construct_invocation_parameters(
+        cls, invocation_parameters: List[InvocationParameterInput]
+    ) -> Dict[str, Any]:
+        supported_params = cls.supported_invocation_parameters()
+        params = {param.invocation_name: param for param in supported_params}
+
+        formatted_invocation_parameters = dict()
+
+        for param_input in invocation_parameters:
+            invocation_name = param_input.invocation_name
+            if invocation_name not in params:
+                raise ValueError(f"Unsupported invocation parameter: {invocation_name}")
+
+            param_def = params[invocation_name]
+            value = extract_parameter(param_def, param_input)
+            if value is not UNSET:
+                formatted_invocation_parameters[invocation_name] = value
+        validate_invocation_parameters(supported_params, formatted_invocation_parameters)
+        return formatted_invocation_parameters
+
+
+@register_llm_client(
+    provider_key=GenerativeProviderKey.OPENAI,
+    model_names=[
+        PROVIDER_DEFAULT,
```

Comment:
it's because not all model names can be registered, Azure requires that users name their own models, which we can't know in advance. Other playground implementations allow users to pass in a model name without it being explicitly on the list (for example, finetuned versions). Passing in "None" is our fallback client for if the model name is not explicitly registered

Reply:
Got it, makes sense.